tests/tracking/test_fluent.py

```diff
@@ -233,137 +232,41 @@ def test_start_run_defaults_databricks_notebook(empty_active_run_stack):
         active_run = start_run()
         MlflowClient.create_run.assert_called_once_with(
             experiment_id=mock_experiment_id,
-            run_name=None,
             tags=expected_tags
         )
         assert is_from_run(active_run, MlflowClient.create_run.return_value)
 
 
-def test_start_run_overrides(empty_active_run_stack):
```

Comment:
Hm why delete these tests?

Reply:
The start_run properties are no longer overridable, as they don't exist